salt/metalk8s/registry/init.sls

```diff
@@ -0,0 +1,49 @@
+{% set registry_image = 'docker.io/registry' %}
+{% set registry_version = '2.7.1' %}
+{% set registry_user = 'metalk8s-registry' %}
+{% set registry_group = 'metalk8s-registry' %}
+
+Inject OCI registry image:
+  containerd.image_managed:
+    - name: docker.io/library/registry:2.7.1
+    - archive_path: /srv/scality/metalk8s-dev/images/registry-2.7.1.tar
+
+Create OCI registry user:
+  group.present:
+    - name: {{ registry_group }}
+    - system: true
+
+  user.present:
+    - name: {{ registry_user }}
+    - fullname: MetalK8s Registry
+    - gid: {{ registry_group }}
```

Comment:
```
gid_from_name
: False
If True, the default group id will be set to the id of the group with the same name as the user. If the group does not exist the state will fail.
```
Less config data ? :stuck_out_tongue:

Reply:
It's not configuration, it's a constant defined at the top of this file.

I don't really like `gid_from_name`: I need to use the group name at other places, and I'd rather be explicit about its value instead of using `gid_from_name` here, and then at the other place remember 'Oh, I need the group name here, but should remember that I should use `registry_user` as its value'.